README.md

```diff
@@ -15,7 +15,16 @@ def deps do
 end
 ```
 
+## Getting Started
+
+Run the mix task to generate the migrations to add the published messages and consumed messages tables to your app's DB:
+
+```bash
+mix generate_railway_migrations
```

Comment:
`mix railway_ipc.generate_migrations`. The namespace might be a nice thing to setup.